### PR TITLE
Increase upload limits

### PR DESCRIPTION
**Pull request by rotated8**
New limits are 100 files, each no larger than 1.2 GB, and no larger than 10.5 GB in total.

### CODE REVIEW

**Comment by rotated8** [comment]
DO NOT MERGE. Waiting for feedback from Shane and/or @zhiwuxie

File: app/assets/javascripts/sufia/uploader.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+//= require fileupload/jquery.iframe-transport
+//= require fileupload/jquery.fileupload.js
+//= require fileupload/jquery.fileupload-ui.js
+//= require fileupload/locale.js
+//
+/*jslint nomen: true */
+/*global $ */
+
+
+//200 MB  max file size
```
Comment (yinlinchen) [inline app/assets/javascripts/sufia/uploader.js]: Should it change to the same value as max_file_size?

File: app/assets/javascripts/sufia/uploader.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+var max_file_size = 1200000000;
+var max_file_size_str = "200 MB";
+//500 MB max total upload size
```
Comment (yinlinchen) [inline app/assets/javascripts/sufia/uploader.js]: Should it change to the same value as max_total_file_size?

**Comment by rotated8** [comment]
Updated comments, and accepted by Shane.

File: app/assets/javascripts/sufia/uploader.js [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+//= require fileupload/jquery.fileupload-ui.js
+//= require fileupload/locale.js
+//
+/*jslint nomen: true */
+/*global $ */
+
+
+//200 MB  max file size
+var max_file_size = 1200000000;
+var max_file_size_str = "200 MB";
```
Comment (pmather) [inline app/assets/javascripts/sufia/uploader.js]: Shouldn't this say "1.2 GB"?

File: app/assets/javascripts/sufia/uploader.js [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+//500 MB max total upload size
+var max_total_file_size = 10500000000;
+var max_file_count = 100;
+var max_total_file_size_str = "500 MB";
```
Comment (pmather) [inline app/assets/javascripts/sufia/uploader.js]: Similarly, shouldn't this say "10.5 GB"?

**Pull request merged**